frame/alliance/src/benchmarking.rs

```diff
@@ -84,7 +84,7 @@ fn generate_unscrupulous_account<T: Config<I>, I: 'static>(index: u32) -> T::Acc
 fn set_members<T: Config<I>, I: 'static>() {
 	let founders: BoundedVec<_, T::MaxMembersCount> =
 		BoundedVec::try_from(vec![founder::<T, I>(1), founder::<T, I>(2)]).unwrap();
```

Comment:
we set only two members for each role here.
for a worst case benches we will need to init Max* members for every role. 
But this does not concern this PR